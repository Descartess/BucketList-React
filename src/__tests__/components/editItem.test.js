import React from 'react';
import renderer from 'react-test-renderer';
import EditBucketListItem from '../../components/editItem';

describe('banner snapshot ', () => {
  const props = {
    onShowEditItem: jest.fn(),
    onToggleCompleted: jest.fn(),
    editBucketListItem: jest.fn(),
    onUsernameChange: jest.fn(),
    item_name: '',
    completed: false,
    showEditItem: false,
  };
  it('renders correctly', () => {
    const rendered = renderer.create(
      <EditBucketListItem {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
