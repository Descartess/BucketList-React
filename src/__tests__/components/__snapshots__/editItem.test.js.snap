// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner snapshot  renders correctly 1`] = `null`;
